api/test/integration/types.test.js

```diff
@@ -0,0 +1,34 @@
+const { describe, test } = require('mocha')
+const { expect } = require('chai')
+
+const { setupApi, runProcess, utf8ToUint8Array, getLastTokenId, getToken } = require('../helper/routes')
+
+const USER_ALICE_TOKEN = '5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY'
+const METADATA_KEY_LENGTH = 32
+const METADATA_VALUE_LITERAL_LENGTH = 32
+
+describe('types', function () {
+  const context = {}
+
+  before(async function () {
+    await setupApi(context)
+  })
+
+  describe('health check', function () {
+    test('mint token', async function () {
+      const lastTokenId = await getLastTokenId(context)
```

Comment:
you can also wrap 

```js
const lastTokenId = await getLastTokenId(context)
```

in the below. It also might be a good idea to assert for more props as at the moment we just check for id

```js
befooreEach(async () => {
  await getLastTokenId(context)
})
```

Reply:
I see the tests as more asserting that we get 'something' rather than the props of that thing. If the types are wrong the whole thing blows up with a decoding error